src/main/protobuf/v1/relationship.proto

```diff
@@ -20,6 +20,9 @@ message PartyRelationshipV1 {
   optional string fileName = 11;
   optional string contentType = 12;
   optional string onboardingTokenId = 13;
+  optional string pricingPlan = 14;
+  optional InstitutionUpdateV1 institutionUpdate = 15;
```

Comment:
here I'm missing the use case: what is the purpose of having the update of an institution in one of its relationships?

Reply:
during an onBoarding, we are asking to admin users to fill some billing and institution info (we are handling also not IPA parties, for which they could also change all the info)
These info will be stored inside the institution if the onboarding is completed

Reply:
I don't know the full picture so I could talk nonsense here, but shouldn't these info be related to the institution party itself instead of one of its relationships? 
What happens if the onboarding is completed for 2 admins at the same time and both of them have this field populated?
Or probably more feasible, being the relationship tied to a specific product, if 2 onboardings are pending for the same institution but different products, and in both cases they fill these infos, but with different data?

Reply:
These info will be filled just for the relationship with the manager (i'm talking about the first onboarding of the product, when storing also the manager)